__tests__/preprocessor.test.js

```diff
@@ -26,6 +26,25 @@ const sources = [
     '../media-box.component.scss',
     './media-box-h0.component.scss'
   ],
+})`,
+  `@Component({
+  selector: 'xc-media-box-h0',
+  templateUrl: 'media-box-h0.component.html',
+  styleUrls: [
+    '../../box.component.scss',
+    '../media-box.component.scss',
+    './media-box-h0.component.scss'
+  ],
+})`,
+  `@Component({
+  selector: 'xc-media-box-h0',
+  templateUrl: 'media-box-h0.component.html',
+  styleUrls: [
+    '../../../box.component.scss',
+    '../../box.component.scss',
+    '../media-box.component.scss',
+    './media-box-h0.component.scss'
+  ],
```

Comment:
Looks like this case is not necessary. Can you remove?

Reply:
Can if you want - it's intention was to make sure I wasn't just making it work for 3 external stylesheets.